.travis.yml

```diff
@@ -33,8 +33,7 @@ addons:
 script: ./travis.sh 
 
 notifications:
-  email:
-    - cukes-devs@googlegroups.com
+  email: false
```

Comment:
For the part to only trigger on a change from failure to success (instead of every success) I think so: https://www.appveyor.com/docs/notifications/#triggering-notifications

But for sending mails to the commit author and committer I cannot find anything in the docs there.